modules/core/src/lib/animation-loop-proxy.js

```diff
@@ -243,9 +247,10 @@ export default class AnimationLoopProxy {
 
   _resizeCanvasDrawingBuffer() {
     if (this.autoResizeDrawingBuffer) {
-      const devicePixelRatio = this.useDevicePixels ? window.devicePixelRatio || 1 : 1;
-      const width = this.canvas.clientWidth * devicePixelRatio;
-      const height = this.canvas.clientHeight * devicePixelRatio;
+      // TODO: get access to gl context and use 'mapToDevicePosition'
```

Comment:
Why do we not have access to the gl context here?

Reply:
Not sure, but yes we might be able to , I will work with @xiaoji on this.

Reply:
Added a TODO , can be addressed in a different PR.